scanner/e2etests/helmchart/templates/scanner-deployment.yaml

```diff
@@ -0,0 +1,58 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.app.scanner.name }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ .Values.app.scanner.name }}
+spec:
+  replicas: 1
+  minReadySeconds: 15
+  selector:
+    matchLabels:
+      app: {{ .Values.app.scanner.name }}
+  template:
+    metadata:
+      namespace: {{ .Release.Namespace }}
+      labels:
+        app: {{ .Values.app.scanner.name }}
+    spec:
+      securityContext:
+        fsGroup: 65534  # Set the group ownership to the `nobody` group, see dockerfile.
+      containers:
+      - name: scanner
+        image: "{{ ( printf "%s/%s:%s" .Values.image.repository .Values.app.scanner.name .Values.image.tag ) }}"
+        imagePullPolicy: IfNotPresent
+        args:
```

Comment:
Let's also explicitly set the command to make it clear what is running

Reply:
9ba32083f8